app/src/main/java/com/locked/shingranicommunity/models/LoginResponse.kt

```diff
@@ -11,4 +11,15 @@ data class LoginResponse(
         return errorType != null || !errors.isEmpty()
 }
 
+}
+
+data class RegisterResponse(
```

Comment:
Can you create a separate file for this

Reply:
I think all data class can be fit in one file if we don't use any other functions or logics

Reply:
The problem is finding the files if you put all of them in the same file. For example the file is called AppModel.kt but it will have Template.kt, Field.kt etc. It's better if they're in separate files so we can easily find them.